SPRINTUSER_README.md

```diff
@@ -57,10 +59,10 @@ Extract the drupal_sprint_package.RELEASE.zip file, and open or browse to the "d
 
 **⚠️Docker for Windows**: You *must* share the C: drive (preferably all drives, or any other drive your home directory may be on) under Docker->Settings->Shared Drives.
 
-**⚠️All users:** additional information is available at [ddev docker instructions](https://ddev.readthedocs.io/en/latest/users/docker_installation/).
```

Comment:
I don't understand the removal of this paragraph - Especially Linux users need more info.